### ISSUE TITLE
[VT-6] vit; cmd o [args]

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#28]
_David Patrick on 2013-04-08T04:25:42Z says:_ 

 The "o" command passes the filter/ uuid set, and any following arguments, to taskopen.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-04-08T04:25:42Z
Modified: 2014-02-09T01:14:58Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-08-14T07:52:11Z says:_ 

 I'm sure you can guess what I'm going to say :)

I'd like to enhance shortcuts to be able to deal with this. Can you give an example of keystrokes in Vit and then what you would want the taskopen command to be?

**Comment by taskwarrior** [comment]
_David Patrick on 2013-08-14T10:53:16Z says:_ 

 Scott Kostyshak wrote:
> I'm sure you can guess what I'm going to say :)
> 
> I'd like to enhance shortcuts to be able to deal with this. 

Absolutely!

> Can you give an example of keystrokes in Vit and then what you would want the taskopen command to be?

Yes and no! It will be easy to do when I'm reconnected with the appropriate hardware, in a day or so.
Thanks for all the enthoosiasm Scott!

**Comment by taskwarrior** [comment]
_David Patrick on 2013-09-15T01:56:50Z says:_ 

 Works with vit 1.2!

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-09-15T06:37:15Z says:_ 

 Fixed at 1da206b8.
